bin/submit_gpu_job

```diff
@@ -13,38 +16,47 @@ TWO_DAYS = 2 * 24 * 60
 def write_slurm_template(script, out_path, arguments,
                          n_threads, gpu_type, n_gpus,
                          mem_limit, time_limit, qos,
-                         mail_address, exclude_nodes):
-    slurm_template = ("#!/bin/bash\n"
-                      "#SBATCH -A kreshuk\n"
-                      "#SBATCH -N 1\n"
-                      "#SBATCH -c %s\n"
-                      "#SBATCH --mem %s\n"
-                      "#SBATCH -t %i\n"
-                      "#SBATCH --qos=%s\n"
-                      "#SBATCH -p gpu\n"
-                      "#SBATCH -C gpu=%s\n"
-                      "#SBATCH --gres=gpu:%i\n") % (n_threads,
-                                                    mem_limit, time_limit,
-                                                    qos, gpu_type, n_gpus)
-    if mail_address is not None:
-        slurm_template += ("#SBATCH --mail-type=FAIL,BEGIN,END\n"
-                           "#SBATCH --mail-user=%s\n") % mail_address
+                         exclude_nodes, array):
+    user = pwd.getpwuid(os.getuid()).pw_name
+    slurm_out = Path(f"/scratch/{user}/slurm_out")
+    slurm_out.mkdir(parents=True, exist_ok=True)
+    slurm_template = (
+        f"#!/bin/bash\n"
+        f"#SBATCH -A kreshuk\n"
+        f"#SBATCH -N 1\n"
+        f"#SBATCH --mail-type=END,FAIL\n"
+        f"#SBATCH --mail-user={user}@embl.de\n"
```

Comment:
I don't really like getting all the spam mail from slurm, this creates a lot of noise when submitting many jobs.
So I would rather leave this optional. In case you are annoyed to always put in the mail address and would rather have this by default, we could add an environment variable to control the behavior, something like:
```
send_mail_by_default = os.environ.get('GPU_ENV_SEND_MAIL', False)
if mail_address is None and send_mail_by_default:
  mail_address = f"{user}@embl.de"
```